Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/kuwo/PlayList.py", line 713, in on_downloaded
    self.do_cache_song_pool()
  File "/usr/lib/python3.6/site-packages/kuwo/PlayList.py", line 731, in do_cache_song_pool
    notify.show()
GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Notifications was not provided by any .service files (2)
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/kuwo/Player.py", line 397, in sync_adjustment
    self.app.lrc.sync_lrc(offset)
  File "/usr/lib/python3.6/site-packages/kuwo/Lrc.py", line 163, in sync_lrc
    timestamp < self.lrc_obj[line_num][0]):
IndexError: list index out of range
/usr/lib/python3.6/site-packages/kuwo/Player.py:768: Warning: Source ID 56585 was not found when attempting to remove it
  GLib.source_remove(self.adj_timeout)
TypeError: <lambda>() takes 2 positional arguments but 4 were given
TypeError: <lambda>() takes 2 positional arguments but 4 were given
TypeError: <lambda>() takes 2 positional arguments but 4 were given
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/kuwo/PlayList.py", line 713, in on_downloaded
    self.do_cache_song_pool()
  File "/usr/lib/python3.6/site-packages/kuwo/PlayList.py", line 731, in do_cache_song_pool
    notify.show()
GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Notifications was not provided by any .service files (2)
TypeError: <lambda>() takes 2 positional arguments but 4 were given
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/kuwo/PlayList.py", line 713, in on_downloaded
    self.do_cache_song_pool()
  File "/usr/lib/python3.6/site-packages/kuwo/PlayList.py", line 731, in do_cache_song_pool
    notify.show()
GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Notifications was not provided by any .service files (2)
TypeError: <lambda>() takes 2 positional arguments but 4 were given
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/kuwo/PlayList.py", line 713, in on_downloaded
    self.do_cache_song_pool()
  File "/usr/lib/python3.6/site-packages/kuwo/PlayList.py", line 731, in do_cache_song_pool
    notify.show()
GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Notifications was not provided by any .service files (2)
TypeError: <lambda>() takes 2 positional arguments but 4 were given
